hack/lib/tracing.bash

```diff
@@ -53,12 +53,12 @@ spec:
               apiVersion: v1
               fieldPath: metadata.namespace
         - name: JAVA_OPTS
-          value: '-Xms128m -Xmx7G -XX:+ExitOnOutOfMemoryError'
```

Comment:
I guess we could avoid the mem if we change the sample rate in Serving and Eventing configs to be `0.1`.
The latter is the value used upstream by default.

Reply:
We need "1.0" and store every single trace. In upgrades it is quite common that only one or two events are not delivered. So we need to trace every event. If we set this to "0.1" we'd not be able to get the required traces most of the time.

Reply:
Ok although we could make this probabilistic in the future.